src/parser/DASHTree.cpp

```diff
@@ -1048,6 +1048,11 @@ static void XMLCALL start(void* data, const char* el, const char** attr)
       }
       else if (strcmp((const char*)*attr, "availabilityStartTime") == 0)
         dash->available_time_ = getTime((const char*)*(attr + 1));
+      else if (strcmp((const char*)*attr, "suggestedPresentationDelay") == 0 ||
+               strcmp((const char*)*attr, "minBufferTime") == 0)
```

Comment:
We probably should ignore this attribute. It's to signal "buffer at least this amount of data before beginning playback" and is not to do with selecting *when* to start. Kodi already has its 8 second internal buffer which is linked to when playback begins.